Automattic/studio
# Studio: Allow import of wp-config.php

Fixes https://github.com/Automattic/dotcom-forge/issues/8661

## Proposed Changes

When a user has a custom table_prefix, this is not currently respected, as we ignore `wp-config.php` file.
This PR solves that.

## Testing Instructions



1. Create a new site.
2. Create a backup from a site with a custom wp_prefix ( You can use the backup located U1WJZT58T/F07GH2T0UAE-slack-file )
3. Import the backup
4. Ensure that the correct tables get used after you start your local site.
5. Check your local site's `wp-config.php` file, and ensure that you see the correct `$table_prefix`.

## Pre-merge Checklist
